spec/que/job_spec.rb

```diff
@@ -83,11 +89,13 @@ def expected_job_count
           enqueue_method.call({'a' => 1, 'b' => 2}, c: 3, d: 4)
           execute
           assert_equal([{a: 1, b: 2}], $args)
+          assert_equal({:c=>3, :d=>4}, $kwargs)
         end
 
         it "should symbolize keyword argument keys" do
           enqueue_method.call(a: 1, b: 2)
           execute
+          assert_equal([], $args)
           assert_equal({a: 1, b: 2}, $kwargs)
```

Comment:
This new assertion doesn't really match the example description (which talks about keyword args). Is this assertion strictly necessary (not redundant) and if so can it be moved into a separate example?

Reply:
With how the args/kwargs separation was messing up, I can see why you'd want to assert on both. We do that in the bulk enqueue spec, just through a helper:

https://github.com/que-rb/que/blob/a82e12eaec8ccf674c16cc9e3344e56900fc83c2/spec/que/job.bulk_enqueue_spec.rb#L77-L78

https://github.com/que-rb/que/blob/a82e12eaec8ccf674c16cc9e3344e56900fc83c2/spec/que/job.bulk_enqueue_spec.rb#L30-L31

Reply:
Yeah, except those examples aren't quite as specific. I'm happy to waive this through but keen to understand whether there is genuinely no redundancy in the tests here or if we're just fleshing them out to consistently make assertions about both `args` and `kwargs`.

Reply:
I can see where you're coming from too. I don't know :shrug: I'm happy either way

Reply:
Yeah I think personally this assertion is just to provide documentation to the reader. It's def not needed but I feel it's nice.